README.md

```diff
@@ -177,8 +172,31 @@ while True:
 c.close()
 ```
 
-See the [examples](examples) directory for more examples, including [how to configure](examples/confluent_cloud.py) the python client for use with
-[Confluent Cloud](https://www.confluent.io/confluent-cloud/).
+**AdminClient**
+
+Create topics:
+
+```python
+    new_topics = [NewTopic(topic, num_partitions=3, replication_factor=1) for topic in topics]
+    # Call create_topics to asynchronously create topics, a dict
+    # of <topic,future> is returned.
+    fs = a.create_topics(new_topics)
+
+    # Wait for each operation to finish.
+    for topic, f in fs.items():
+        try:
+            f.result()  # The result itself is None
+            print("Topic {} created".format(topic))
+        except Exception as e:
+            print("Failed to create topic {}: {}".format(topic, e))
+```
+
+
+
+Thread Safety
```

Comment:
I'm not sure this is the place, better off having this in the API docs for each client type.

Reply:
I think this should definitely be on the front page - it seems an important thing people want to check off, and it adds to the vibe of credibility, which we want to be getting across up front.

Reply:
its also on the front page of other clients, and people incorrectly interpret it as a differentiator for them.